Cargo.lock

```diff
@@ -4172,6 +4237,12 @@ dependencies = [
  "tiny-skia-path",
 ]
 
+[[package]]
+name = "utf8parse"
+version = "0.2.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "06abde3611657adf66d383f00b093d7faecc7fa57071cce2578660c9f1010821"
+
```

Comment:
This adds a lot of dependencies - is there a way to avoid them with feature-flags on `env_logger`, or by using an alternative crate?

Reply:
I don't know. But given that these are examples and tests (things which are rarely compiled in full, or rarely compiled at all, especially compared to `egui` itself), I don't think the extra dependencies matter.